student/src/main/java/tds/student/web/handlers/TestShellHandler.java

```diff
@@ -231,31 +236,7 @@ private String parseOutLatencies (Map<String, String> formParams) {
 
     TestOpportunity testOpp = StudentContext.getTestOpportunity ();
 
-    // get responses
```

Comment:
Right - what @millerts said.  In order to maintain some consistency with how we're handling the switches for legacy/remote calls, the legacy "review test" logic had to be pulled into its own service.